modules/ossm-configuring-jaeger-v1x.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v1x/ossm-custom-resources.adoc
+
+[id="ossm-configuring-jaeger_{context}"]
+= Configuring Jaeger
+
+When the {ProductShortName} Operator creates the `ServiceMeshControlPlane` resource it can also create the resources for distributed tracing. {ProductShortName} uses Jaeger for distributed tracing.
+
+You can specify your Jaeger configuration in either of two ways:
+
+* Configure Jaeger in the `ServiceMeshControlPlane` resource. There are some limitations with this approach.
+
+* Configure Jaeger in a custom `Jaeger` resource and then reference that Jaeger instance in the  `ServiceMeshControlPlane` resource. If a Jaeger resource matching the value of `name` exists, the control plane will use the existing installation. This approach lets you fully customize your Jaeger configuration.
```

Comment:
Based on the previous doc, it suggests for 1.x SMCP it is not based on jaeger 'name', but using URLs?

Reply:
The new mechanism is only supported in 2.0.  For 1.x, users will need to continue setting these fields directly.

Reply:
I think this functionality may be broken in 2.0.  Looking at the code, I don't see how these are mapped into the v2 version of the resource, or vice versa.

Reply:
@rcernich , I assume that since https://github.com/maistra/istio-operator/pull/629 that we can move forward on this PR again?

Reply:
Yes, and the v2 config should work for both 1.x and 2.x control planes.